sandy/sections/mf8.py

```diff
@@ -663,16 +668,48 @@ def _write_rdd(sec):
                         0,
                         list(discr.values())[1:],
                         )
-                if spectra['LCON'] != 0 and 'CONT' in list(sec['SPECTRA'][STYP].keys()):
-                    for RTYP, cont in spectra['CONT'].items():
-                        lines += sandy.write_tab1(
-                            cont['RTYP'],
-                            0.0,
-                            0,
-                            cont['LCOV'],
-                            cont['NBT'],
-                            cont['INT'],
-                            cont['E'][0],
-                            cont["RP"],
-                            )
-        return "\n".join(sandy.write_eol(lines, sec["MAT"], 8, sec["MT"]))
+            if spectra['LCON'] != 1 and 'CONT' not in list(sec['SPECTRA'][STYP].keys()) and 'ER' not in list(sec['SPECTRA'][STYP].keys()):
+                add = [spectra['FD'],
+                       spectra['DFD'],
+                       spectra['ERAV'],
+                       spectra['DERAV'],
+                       spectra['FC'],
+                       spectra['DFC'],
+                       ]
+                lines += sandy.write_list(
+                    0,
+                    STYP,
+                    spectra['LCON'],
+                    0,
+                    0,
+                    add
+                    )
+            if spectra['LCON'] != 0 and 'CONT' in list(sec['SPECTRA'][STYP].keys()):
```

Comment:
These if clauses do not cover all combinations...they could fail for some specific nuclides.
Can you cover all combinations and raise NotimplementedError when not implemented?

Can you test write_rdd for all nuclides in jeff33? (Just in a notebook, not here because it's too long).
You can use `sandy.get_endf6_file("jeff_33", "decay", "all")` implemented by @AitorBengoechea .